### DISCUSSION TITLE
The way to get connected/disconnected status when web3modal open

### DISCUSSION THREAD

**Discussion by ninjacrypto** [WalletConnect/walletconnect-monorepo#3875] [Developer Support] [answered]
I am using web3modal, ethereumclient from cdn. When I used web3modal before.. it's returned a connected provider when open modal. But for now.. I can not get any connected/disconnected status when open modal. I want to wait until user connect with a wallet and return the status from web3modal. Is there a way for this feature?

**Comment by glitch-txs** [comment, marked as answer]
Hi, you can use [Wagmi actions](https://wagmi.sh/core/actions/watchAccount)

**Comment by ninjacrypto** [comment]
Great!, it works well using wagmi watchAccount action to detect the connected status of web3modal.
